src/utils/geom.js

```diff
@@ -20,14 +20,20 @@ export const getShapeFromGeomType = geomType => {
       return 'line';
     case GEOM_UNDEFINED:
       return null;
+    case undefined:
+    case null:
+      return undefined;
     default:
       return 'fill';
   }
 };
 
 export const getLayerStyleDefaultValue = (color, type) => {
   if (!type) {
-    return {};
+    return undefined;
+  }
+  if (type === null) {
+    return null;
```

Comment:
If the `type` parameter equals to `null`, this function should return `undefined`
Maybe you should reverse these two conditions.

Reply:
No I want undefined in all case. I forget to delete this code.